pkg/expr/func.go

```diff
@@ -31,16 +32,21 @@ type Func interface {
 
 	// ArgTypes specifies the argument types in order expected by the function.
 	ArgTypes() []config.ValueType
+}
 
-	// NullArgs specifies if the function accepts null args
-	AcceptsNulls() bool
+// Func implements a function call.
+// It needs to know details about Expressions and attribute bag.
+// Func is performant.
```

Comment:
What does "Func is performant" mean? Seems like an odd statement in an API comment.